frontend/cypress/integration/admin/admin-workflows.spec.js

```diff
@@ -0,0 +1,99 @@
+import { aliasMutation, aliasQuery } from "../../support/graphql-test-utils";
+
+describe("Admin Smoke Tests", () => {
+  const loginAndGoToDashboard = () => {
+    cy.login("admin");
+    cy.visit("/en/admin");
+
+    // make sure we end up on the dashboard
+    cy.findByRole("heading", { name: /Welcome back/i })
+      .should("exist")
+      .and("be.visible");
+    cy.url().should("contain", "/dashboard");
+  };
+
+  beforeEach(() => {
+    cy.intercept("POST", "/graphql", (req) => {
+      aliasQuery(req, "AllUsers");
+      aliasQuery(req, "User");
+      aliasMutation(req, "UpdateUserAsAdmin");
+    });
+
+    loginAndGoToDashboard();
+  });
+
+  it("Searches for a user and reviews the profile", () => {
+    // find the applicant user to review
+    cy.findByRole("link", { name: /Manage users/i }).click();
+    cy.wait("@gqlAllUsersQuery");
+    cy.findByRole("textbox", { name: /Search/i })
+      .clear()
+      .type("Applicant");
+
+    // wait for table to rerender
+    cy.findByText("applicant@test.com")
+      .should("exist")
+      .and("be.visible");
+
+    cy.findByRole("table")
+      .findByRole("row", { name: /applicant/i })
+      .findByText("applicant@test.com") // findByRole link doesn't work here
+      .click();
```

Comment:
Tried running these, and getting a failure. Seems that chaining requires using cypress' `.within()` command:
https://github.com/testing-library/cypress-testing-library/issues/132#issuecomment-853297907

Testing out if this fixes my failure. A bit unsure why it was working for you and not me 🤔

Reply:
Ah wait, there's something wrong and I think I need to refresh_all. So don't dive in here :)

Reply:
as mentioned in slack last week (i think?) findByRole is a bit wonky, so maybe worth abandoning it in places where it gives any trouble. still new to cypress-testing-library, but have only seen this wonkiness within tables tbh, so might be small issue